career_development/ladder.md

```diff
@@ -57,7 +57,8 @@
 -[ ] Can mock http requests and understands where these tests will potentially fail.
 -[ ] Can integrate CI.	
 #### iterative	
--[ ] works with red-green-refactor, actively uses pomodoros,
+-[ ] Works with red-green-refactor.
+-[ ] Actively uses pomodoros.
```

Comment:
Might call out `time management techniques` here instead of a specific technique.